userguide/virtualmachines.rst

```diff
@@ -663,107 +663,88 @@ Start the Docker Host
-
-Continuing to set up the Rancher Host is done from the command line.
-Enter *rancher* as the username, press :kbd:`Enter`, then enter either
-the default password *docker* or the custom password created by editing
-the raw file. Press :kbd:`Enter` again. After logging in, a
-:literal:`[rancher@ClientHost ~]$` prompt is displayed.
+In the serial shell, type :literal:`rancher` for the username and press
+:kbd:`Enter`. Enter either the default password :literal:`docker` or the
+custom password created by editing the raw file and press :kbd:`Enter`.
+A :literal:`[rancher@ClientHost ~]$` prompt will appear.
 
 Ensure Rancher has functional networking and can :command:`ping` an
-outside website. Adjust the VM
-:ref:`Network Interface <vms-network-interface>` and reboot the VM
-if necessary.
-
-Download and install the Rancher system with this command:
+outside website.
 
 .. code-block:: none
 
-   sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server
+   [rancher@ClientHost ~]$ ping -c 5 8.8.8.8
+   PING 8.8.8.8 (8.8.8.8): 56 data bytes
+   64 bytes from 8.8.8.8: seq=0 ttl=121 time=17.466 ms
+   64 bytes from 8.8.8.8: seq=1 ttl=121 time=17.502 ms
+   64 bytes from 8.8.8.8: seq=2 ttl=121 time=17.472 ms
+   64 bytes from 8.8.8.8: seq=3 ttl=121 time=17.426 ms
+   64 bytes from 8.8.8.8: seq=4 ttl=121 time=17.343 ms
 
+   --- 8.8.8.8 ping statistics ---
+   5 packets transmitted, 5 packets received, 0% packet loss
+   round-trip min/avg/max = 17.343/17.441/17.502 ms
 
-.. note:: If the error :literal:`Cannot connect to the Docker daemon`
-   is shown, run :command:`sudo dockerd`. Then give the
-   :command:`sudo docker run` command above again.
 
+If :command:`ping` returns an error, adjust the VM
+:ref:`Network Interface <vms-network-interface>` and reboot the VM.
 
-Installation time varies with processor and network connection speed,
-but typically takes a few minutes. After the process finishes and a
-command prompt is shown, type this command:
-
-.. code-block:: none
-
-   ifconfig eth0 | grep 'inet addr'
+Download and install the Rancher system with
+:command:`sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server`.
 
+If :literal:`Cannot connect to the Docker daemon` shows, run
+:command:`sudo dockerd` and try the command again. Installation time
+varies with processor and network connection speed.
 
-The first value is the IP address of the Rancher server. Enter the IP
-address and port :literal:`8080` as the URL in a web browser. For
-example, if the IP address was :literal:`10.231.3.208`, enter
-:literal:`10.231.3.208:8080` as the URL in the web browser.
+When the command prompt is available, enter
+:command:`ifconfig eth0 | grep 'inet addr'`. Copy the
+:literal:`inet addr:` IP address. Enter the IP address appended with
+:literal:`:8080` into a web browser to connect to the Rancher server
+|web-ui|. For example, if the IP address was :literal:`10.231.3.208`,
```

Comment:
|web-ui| is a FreeNAS-specific term.  Don't use that substitution for other things.  Here, just use a plain "web interface".

"was" is past tense, use "is".